_includes/head.html

```diff
@@ -34,7 +34,7 @@
 <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">
 
 <!-- Code Syntax Highlighting -->
-<link rel="stylesheet" href="https://gitcdn.link/repo/jwarby/jekyll-pygments-themes/master/{{ site.highlight_theme }}.css" />
+<link rel="stylesheet" href="https://gitcdn.link/repo/jwarby/jekyll-pygments-themes/master/{{ site.highlight_theme_light }}.css" media="none" id="highlight_theme_light" />
```

Comment:
since your changes touch this line, do you mind committing these CSS files to `assets/css/` instead and loading them from a relative path? there have been issues with stability of `gitcdn.link` and `gitcdn.xyz` in different geographies (see #367 and #393).